Features/ErrorViews/ErrorView.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  ErrorView.swift
+//  Movies
+//
+//  Created by Diana Chizhik on 18.07.22.
+//
+
+import UIKit
+
+protocol ErrorViewDelegate: AnyObject {
+    func dismissController()
+}
+
+class ErrorView: UIView {
+    lazy var imageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        imageView.heightAnchor.constraint(equalToConstant: 28).isActive = true
+        return imageView
+    }()
+    
+    lazy var errorLabel: UILabel = {
+       let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.font = UIFont.systemFont(ofSize: 15)
+        label.textColor = UIColor(named: "errorTextColor")
+        label.textAlignment = .center
+        label.numberOfLines = 0
+        return label
+    }()
+    
+    lazy var dismissButton: UIButton = {
+       let button = UIButton()
+        let attributes: [NSAttributedString.Key : Any] = [
+            .foregroundColor : UIColor(named: "dismissButtonColor")!,
+            .font : UIFont.systemFont(ofSize: 15)
+        ]
+        let title = NSAttributedString(string: "Dismiss", attributes: attributes)
+        button.setAttributedTitle(title, for: .normal)
+        button.addTarget(self, action: #selector(dismissController), for: .touchUpInside)
+        return button
+    }()
+    
+    lazy var stackView: UIStackView = {
+        let view = UIStackView()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.axis = .vertical
+        view.spacing = 24
+        view.alignment = .center
+        view.layoutMargins = UIEdgeInsets(top: 24, left: 32, bottom: 16, right: 32)
+        view.isLayoutMarginsRelativeArrangement = true
+        view.backgroundColor = UIColor(named: "errorViewBackgroundColor")
+        view.layer.cornerRadius = 5
+        view.layer.masksToBounds = true
```

Comment:
Curiosity: In the past (before iOS 14) this wouldn't work. UIStackViews are purely structural meaning you can only use them for arranging views. Under the hood it used to be a `CATransformLayer`. After iOS 14 it got changed to a `CALayer` which can now handle UI properties. More [here](https://useyourloaf.com/blog/stack-view-background-color-in-ios-14/).

Reply:
I feel lucky to have all this now :)
So, back then I'd have to use UIStackView with another  UIView underneath it?